lib/process-services/src/lib/process-list/components/start-process.component.scss

```diff
@@ -51,14 +51,14 @@
     }
 
     &-start-form-container {
-        .mat-card {
-            box-shadow: none !important;
-            padding: 0 !important;
+        .mat-card.mat-card.mat-card {
```

Comment:
General comment - could you please check all places where you used material classes? 
We have raised task to remove all references to material classes as they should not be used anymore because they block angular upgrade. 
That work for ADF was done by @DominikIwanek so you can contact with him for better alternative. Here is task link: https://alfresco.atlassian.net/browse/ACS-6140

Reply:
just checked and fixed all of the places with reference to Angular Material